### ISSUE TITLE
Disable tab animations

### ISSUE THREAD

**Issue by YarnSphere** [kurogetsusai/firefox-gnome-theme#38]
GNOME applications seem to have no tab animations (when creating a new tab or closing a tab); I believe it's in this theme's interest to copy that.

There used to be an `about:config` option to exclusively disable tab animations (`browser.tabs.animate`) but I cannot find it anymore (using FF 60.0). Currently, disabling `toolkit.cosmeticAnimations.enabled` seems to work, but I believe it is also turning off other animations.

It would be nice if this theme solved this without having to mess around in `about:config`. What do you think? Is it possible to disable the tab animations in the CSS?

If not, perhaps it would be a good idea to add a list of `about:config` settings to the README that further improve this theme's resemblance to a native GNOME application.

Thank you for your work so far!

**Comment by lunakurame** [comment]
I don't think it would be possible to just disable tab animations with our CSS styles, the tab bar has a lot of JS code always trying to mess with our styles: #10 and seems like those animations have something to do with e10s which also causes other bugs: #33. Disabling `toolkit.cosmeticAnimations.enabled` seems to also disable the refresh button animations (ignoring `browser.stopReloadAnimation.enabled`) but not the downloads button ones... which can be disabled with `browser.download.animateNotifications`. Aside from that inconsistency, isn't disabling all of them a desirable effect if you're aiming at more strict GNOME experience?

And yes, I'll add a list of related `about:config` settings to the README, that sounds like a good idea.

**Comment by YarnSphere** [comment]
Sounds like it might be desirable, yes; though GNOME has _some_ animations. For example, menu pop-ups are animated. This might actually give you a better base to copy GNOME's behaviours: since the setting seems to disable most animations, if wanted, you might be able to use custom CSS animations on top of the non-animated UI to copy GNOME.

I'll await your list of `about:config` settings! Maybe you could even provide a default `user.js` file with all of them active?
